src/phpDocumentor/ConfigurationFactory.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+namespace phpDocumentor;
+
+final class ConfigurationFactory
+{
+    /**
+     * @var Uri
+     */
+    private $uri;
+
+    /**
+     * @var string
+     */
+    private $schemaPath;
+
+    /**
+     * @param Uri    $uri
+     * @param string $schemaPath
+     */
+    public function __construct(Uri $uri, $schemaPath = '../../data/xsd/phpdoc.xsd')
```

Comment:
a path is relative to the current working directory. always prefix with the __DIR__ constant if you intend to indicate a path relative to the current file